packages/turbo-types/package.json

```diff
@@ -14,22 +14,20 @@
   },
   "main": "src/index.ts",
   "types": "src/index.ts",
-  "bin": {
-    "turbo-types-generate": "./src/scripts/codegen.js"
-  },
   "scripts": {
+    "build": "tsc && pnpm generate-schema",
```

Comment:
We need to update the `outputs` for `@turbo/types#build` to include `schemas` otherwise they won't be present on cache hits.

Reply:
added in a3dbcc7ac20d532b367f3d3f31a04778cff69195